### PR TITLE
Issue 800 update filter housing docs

### PR DESCRIPTION
**Pull request by RakurthiLokesh**

### CODE REVIEW

File: manufacturing/filter_holder/README.md [hunk 1]
```diff
@@ -42,13 +52,15 @@ We may have parts to send you from our inventory.
 | 6    | ~4 sq in | McMaster-Carr | Gasket Material      | $22.22/ 12x12" sheet | [C][9mcmc] | cut with jigs to make items 4 and 5 |
 | 7    |        2 | McMaster-Carr | 96016A556            | $5.50/10pk       | [C][7mcmc] | thumbscrews |
 | 8    |        2 | McMaster-Carr | 94180A333            | $16.69/100pk     | [C][8mcmc] | thumbscrews |
+| 9    |        1 | RespiraWorks  | Filter Non-patient Outer Housing | $1 (material)    | [B][9rw]   | 3D printed from data in link |
```
Comment (martukas) [inline manufacturing/filter_holder/README.md:55]: I would move this to right after items 1 and 2.

Comment (martukas) [reply manufacturing/filter_holder/README.md:55]: And the quantities should probably be reflected for the entire ventilator. So 2x of these, and 2x of 1, 4x of 2? Does that make sense?

Comment (RakurthiLokesh) [reply manufacturing/filter_holder/README.md:55]: yeah it make sense and will update that.

File: manufacturing/filter_holder/README.md [hunk 2]
```diff
@@ -34,22 +44,26 @@ We may have parts to send you from our inventory.
...
 | ---- |---------:| ------------- | -------------------- | ----------------:|:-----------|:------|
-| 1    |        1 | RespiraWorks  | Filter Outer Housing | $1 (material)    | [B][1rw]   | 3D printed from data in link |
-| 2    |        1 | RespiraWorks  | Filter Inner Housing | $1 (material)    | [B][2rw]   | 3D printed from data in link |
-| 3    |        1 | Roomba        | Filter               | $8/6pk           | [Z][3amzn] | removed from filter unit as delivered |
-| 4    |        1 | RespiraWorks  | Filter Inner Gasket  | $0.10 (material) | B          | make using item 6 below |
-| 5    |        1 | RespiraWorks  | Filter Outer Gasket  | $0.10 (material) | B          | make using item 6 below |
-| 6    | ~4 sq in | McMaster-Carr | Gasket Material      | $22.22/ 12x12" sheet | [C][9mcmc] | cut with jigs to make items 4 and 5 |
-| 7    |        2 | McMaster-Carr | 96016A556            | $5.50/10pk       | [C][7mcmc] | thumbscrews |
-| 8    |        2 | McMaster-Carr | 94180A333            | $16.69/100pk     | [C][8mcmc] | thumbscrews |
+| 1    |        2 | RespiraWorks  | Filter Outer Housing | $1 (material)    | [B][1rw]   | 3D printed from data in link |
```
Comment (martukas) [inline manufacturing/filter_holder/README.md]: And I guess this should then say "patient outer housing"

Comment (martukas) [reply manufacturing/filter_holder/README.md]: and maybe switch with Item 2 below.

File: manufacturing/filter_holder/README.md [hunk 3]
```diff
@@ -34,22 +44,26 @@ We may have parts to send you from our inventory.
...
+| 2    |        2 | RespiraWorks  | Filter Inner Housing | $1 (material)    | [B][2rw]   | 3D printed from data in link |
```
Comment (martukas) [inline manufacturing/filter_holder/README.md:48]: and this one should be quantity 4

File: manufacturing/filter_holder/README.md [hunk 4]
```diff
@@ -34,22 +44,26 @@ We may have parts to send you from our inventory.
...
+| 3    |        2 | RespiraWorks  | Filter Non-patient Outer Housing | $1 (material)    | [B][3rw]   | 3D printed from data in link |
+| 4    |        1 | Roomba        | Filter               | $8/6pk           | [Z][4amzn] | removed from filter unit as delivered |
+| 5    |        4 | RespiraWorks  | Filter Inner Gasket  | $0.10 (material) | B          | make using item 6 below |
+| 6    |        4 | RespiraWorks  | Filter Outer Gasket  | $0.10 (material) | B          | make using item 6 below |
+| 7    | ~4 sq in | McMaster-Carr | Gasket Material      | $22.22/ 12x12" sheet | [C][9mcmc] | cut with jigs to make items 4 and 5 |
+| 8    |        2 | McMaster-Carr | 96016A556            | $5.50/10pk       | [C][7mcmc] | thumbscrews |
```
Comment (martukas) [inline manufacturing/filter_holder/README.md:54]: And these last two should probably be quantity 8 now?

File: manufacturing/filter_holder/README.md (hunk 2 above)
Comment (RakurthiLokesh) [reply manufacturing/filter_holder/README.md]: i have doubt with the names, the panel mounting part is called as outer and the one to which patient circuit is connected is called inner, can you give me clarity about that?..

File: manufacturing/filter_holder/README.md [hunk 5]
```diff
@@ -30,26 +40,31 @@ BEFORE purchasing any parts.**
 
 **NOTE:** Before you buy anything on this list, check with Edwin, Martin or Ethan. Blower takes a while to deliver.
```
Comment (martukas) [inline manufacturing/filter_holder/README.md:41]: Actually this stuff about blower is irrelevant. Maybe we can remove it? Only the note you added should be here.

File: manufacturing/filter_holder/README.md (hunk 2 above)
Comment (martukas) [reply manufacturing/filter_holder/README.md]: I think the better terminology should be:
"outer housing with patient tubing interface"
and 
"outer housing without tubing interface"

**Pull request merged**